lighthouse-core/gather/gather-runner.js

```diff
@@ -631,6 +631,11 @@ class GatherRunner {
 
       await GatherRunner.setupDriver(driver, options);
 
+      const milestone = (await driver.getBrowserVersion()).milestone;
+      if (milestone < 83) {
```

Comment:
Typically we place these checks closer to the intended use. (Someone using LH without the perf category doesn't have any problem)

@connorjclark so i'm thinking this could go in either the LCP audit itself or trace-processor. wdyt?


Also.. Oddly, our existing comment sez 78, but the bug reporter also had 78.
https://github.com/GoogleChrome/lighthouse/blob/b8bc05aa669913fdc5a7eb4f11ee1b9dc229d832/lighthouse-core/computed/metrics/largest-contentful-paint.js#L10

 So i suppose the 78 is wrong, but what about 79-82?

Reply:
> Typically we place these checks closer to the intended use. (Someone using LH without the perf category doesn't have any problem)

I was too late with my comment, agree with ☝️ 

> this could go in either the LCP audit itself or trace-processor

my vote is definitely in LCP as it keeps it local to what cares the most about whether or not there was an LCP trace event. `trace-processor` could in theory be looking at something that's not expected to have an LCP event and it's no big deal.

<hr>

But @paulirish I take it you want to abandon the "Add toplevelwarning about old Chrome" goal? I ask because you were the one to change the title of #10499 to that, but I would guess it was during a bug bash and we were all there :)

Reply:
In the LCP audit it looks like we can fetch the chrome version with `artifacts.HostUserAgent` but that returns an unfriendly string like `Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/83.0.4103.116 Safari/537.36`.  Is there a better way to fetch the chrome milestone in the audit?

Reply:
@adamraine good call. right now that's the best we got. But on the bright side, we know it's a Chrome UA, so there's not much variability we have to deal with. So a pretty simple regex should do the job.

Reply:
```
'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/83.0.4103.116 Safari/537.36'.match(/Chrome\/(\d+)/)[1]
```

fyi